### PR TITLE
feat: 421 enforce request limit

### PR DESCRIPTION
**Pull request by yetti**

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -106,6 +106,14 @@ class CatalogController < ApplicationController
...
 
     config.add_facet_field "format", label: "Format", limit: 20
+    config.add_facet_field "author_ssim", label: "Author", limit: true, index_range: "A".."Z"
+    config.add_facet_field "subject_ssim", label: "Subject", limit: 20, index_range: "A".."Z"
+    config.add_facet_field "language_ssim", label: "Language", limit: true
+    config.add_facet_field "austlang_ssim", label: "Aboriginal and Torres Strait Islander Language", limit: 10
+    config.add_facet_field "lc_1letter_ssim", label: "Call Number"
+    config.add_facet_field "geographic_name_ssim", label: "Geographic", limit: true, index_range: "A".."Z"
+    config.add_facet_field "series_ssim", label: "Series", limit: true, index_range: "A".."Z", single: true
+    config.add_facet_field "subject_era_ssim", label: "Era"
```
Comment (scott-yeadon) [inline app/controllers/catalog_controller.rb:116]: what is subject_era, I don't have that in the index

File: app/views/account/requests.html.erb [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
+<<<<<<< HEAD
+  <div class="alert alert-danger" role="alert">
+    <%= t("requesting.request_limit_error", url: ENV["ASK_LIBRARIAN_URL"]).html_safe %>
+  </div>
+=======
+<div class="alert alert-danger" role="alert">
+  <%= t("requesting.request_limit_error", url: ENV["ASK_LIBRARIAN_URL"]).html_safe %>
+</div>
+>>>>>>> dceac25 (feat: implement request limit error message)
+<%- else %>
```
Comment (scott-yeadon) [inline app/views/account/requests.html.erb:12]: unmerged?

File: app/views/request/new.html.erb [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
 
+<%- if @met_request_limit %>
+  <div class="alert alert-danger" role="alert">
+    <%= t("requesting.request_limit_error", url: ENV["ASK_LIBRARIAN_URL"]).html_safe %>
+  </div>
+<%- end %>
+<<<<<<< HEAD
+=======
+
+>>>>>>> dceac25 (feat: implement request limit error message)
```
Comment (scott-yeadon) [inline app/views/request/new.html.erb:11]: remnants of merge?

**Review by scott-yeadon — APPROVED** [review]

**Comment by yetti** [comment]
@scott-yeadon GDI, I've had so many problems merging changes into this branch. I'll fix it up and re-open this PR when it's ready.

**Pull request closed without merging**